lib/support/windows.js

```diff
@@ -92,13 +92,19 @@ const WindowsSupport = {
   },
 
   arch() {
-    if (!this.cachedArch) { this.cachedArch = os.arch(); }
+    /* if (!this.cachedArch) { this.cachedArch = os.arch(); }
```

Comment:
Is there a reason we're commenting this out instead of just deleting it?

Reply:
to keep a reminder in the code for something we may want to revert back to, at least in some form.